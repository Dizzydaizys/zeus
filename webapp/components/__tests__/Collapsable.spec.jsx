import React from 'react';
import {render} from 'enzyme';

import Collapsable from '../Collapsable';

describe('Collapsable', () => {
  it('renders collapsed', () => {
    const tree = render(
      <Collapsable maxVisible={2} collapsable={true}>
        <div>1</div>
        <div>2</div>
        <div>3</div>
        <div>4</div>
      </Collapsable>
    );
    expect(tree).toMatchSnapshot();
  });
});
