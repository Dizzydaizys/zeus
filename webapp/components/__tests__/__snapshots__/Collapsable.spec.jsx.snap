// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Collapsable renders collapsed 1`] = `
<div>
  <div>
    1
  </div>
  <div>
    2
  </div>
  <a
    class="sc-gzVnrw fAqkiO"
  >
    <div
      class="sc-bZQynM kdaNqr ResultGrid-Row sc-htpNat iSNGhk sc-bdVaJa iHZvIS"
      style="color:inherit"
    >
      <div
        class="sc-ifAKCX gVDkew ResultGrid-Column sc-bdVaJa kYmYWE"
      >
        Show 2 other item(s)
      </div>
    </div>
  </a>
</div>
`;
